### DISCUSSION TITLE
Exception ao tentar enviar email.

### DISCUSSION THREAD

**Discussion by oGrandeK** [balta-io/forum#279] [Tira Dúvidas] [answered]
Olá professor, estava recriando seus passos no módulo de enviar email e acontece que no meu código está dando essa Exception: "Unable to read data from the transport connection: The connection was closed."

O que devo fazer?

**Comment by oGrandeK** [comment, marked as answer]
Andei fuçando a internet e vi que a porta do sendgrid tinha mudado, fiz todas as alterações e agora é outra exception: "The operation has timed out".

**Reply by andrebaltieri** [reply]
Boa tarde, @oGrandeK.. como vai?

Será que não tem nenhum bloqueio no seu computador? As duas mensagens são erros de rede. Consegue testar de outra máquina, em outra rede?

**Reply by oGrandeK** [reply]
Tinha sim, desativei o firewall do windows e consegui fazer alguns envios mudando algumas portas no código, no final das contas deu certo kkkkkk

**Reply by andrebaltieri** [reply]
Show!!!!
